packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -138,7 +144,49 @@ const ModalContent = forwardRef((props, ref) => {
   delete customProps.closePortal;
   delete customProps.fallbackFocus;
 
-  const platformIsiOS = !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);
+  const modalContent = (
+    <div
+      {...customProps}
+      aria-label={ariaLabel}
+      aria-labelledby={ariaLabelledBy}
+      aria-describedby={ariaDescribedBy}
```

Comment:
We need to validate with screen readers to ensure values provided to above aria attributes are read when the focus is on modal container. Since there is no tabIndex on this div, it may get ignored by screen readers

Reply:
Hi Supreeth, when the focus is on model container all the aria attributes are working as expected. Please find the attached video for reference.

https://github.com/cerner/terra-framework/assets/126655256/a525f425-680b-4254-8818-08cd568d131f

![Uploading Screenshot 2024-02-15 at 2.25.43 PM.png…]()

Reply:
Please verify on JAWS as well

Reply:
In JAWS as well It is announcing all the aria attributes PFA video


https://github.com/cerner/terra-framework/assets/126655256/b8ca364b-948f-4b05-b1b3-a8c67cee2e90